### ISSUE TITLE
Remove ABI folder

### ISSUE THREAD

**Issue by schmidsi** [melonproject/melon.js#128]
ABIs should be directly imported from the protocol. Therefore, this folder is obsolete: https://github.com/melonproject/melon.js/tree/parityExp/abi

If this was only a temporary hack, see this ticket as a reminder to remove this folder.

**Comment by jennazenk** [comment]
Yes ABI folder is temporary until the protocol is deployed with the ABI as .json

**Comment by jennazenk** [comment]
But folder is not obsolete yet - will be on next deployment hopefully
